include/boost/winapi/process.hpp

```diff
@@ -428,7 +455,11 @@ BOOST_FORCEINLINE BOOL_ CreateProcessW(
     INT_ bInheritHandles,
     DWORD_ dwCreationFlags,
     LPVOID_ lpEnvironment,
-    LPCWSTR_ lpCurrentDirectory,
+#if defined(_WIN32_WCE)
+    LPWSTR_ lpCurrentDirectory,
+#else
+	LPCWSTR_ lpCurrentDirectory,
```

Comment:
No tabs, please. Here and everywhere else.

Reply:
agree